person.rb

```diff
@@ -1,17 +1,19 @@
 require './nameable'
 require './capitalize_decorator'
 require './trimmer_decorator'
+require './rental'
 
 class Person < Nameable
```

Comment:
It will be great to have a method that adds the rentals. Remember there is a many-to-many relationship between a person and a book through rentals. Hence there should be a method of adding the rentals in this class. Example:

```ruby
  def add_rental(book, date)
    # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    Rental.new(date, book, self)
  end
```